pkg/minishift/docker/image/sysproc_linux.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
```

Comment:
True, but it belongs to image_handler. If I would not have to handle the different OS types specifically, I would not have pulled this out. I guess it be moved into something like _pkg/os/process_. I can see this used for sshfs host folders as well.